decentraland/unity-renderer
# Fix: fixed broken scene text shape

## What does this PR change?

Fix #2893 
Removes text shape minor optimization in order to fix the broken texts

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/broken-text-shape
2. Go to an ice poker scene
3. Play (ask Kyllian if you need a wearable to test the scene)
4. Check that all the texts appear correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
